### PR TITLE
UML-1213: added access code viewed date by organisations

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-1213

Fixes UML-1213

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

**Comment by cooperaj** [comment]
If there have been no updated translations provided (i.e. welsh) then there is no need to generate new .po or .mo files as they provide no benefit and may confuse future translation work. Should probably remove those files from this PR.

**Comment by SeemaMenon** [comment]
> If there have been no updated translations provided (i.e. welsh) then there is no need to generate new .po or .mo files as they provide no benefit and may confuse future translation work. Should probably remove those files from this PR.

@cooperaj  - how about the "on %date%" in LPA Viewed field? Thought that needs to be within the translations tag.
Screenshot 2021-02-26 at 10 43 46

File: service-front/app/src/Actor/templates/actor/partials/check-code-details.html.twig [hunk 1]
```diff
@@ -40,7 +40,7 @@
                         {% else %}
                             {% for code in code.Viewed %}
                                 {% if code.ViewedBy is not empty %}
-                                    {{ code.ViewedBy }}<br>
+                                    {{ code.ViewedBy }} {% trans with {'%date%': code_date(code.Viewed, 'Y-m-d\\TH:i:s.u\\Z')} %}on %date%{% endtrans %}<br>
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/partials/check-code-details.html.twig:43]:
```suggestion
                                    {% trans with {'%organisation%': code.ViewedBy, '%date%': code_date(code.Viewed, 'Y-m-d\\TH:i:s.u\\Z')} %}%organisation% on %date%{% endtrans %}<br>
```

We want to ensure the full statement is to be translated. If the welsh expects a different order i.e. %date% by %name% then we can't do that if we hard code the order in the templates.

Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/partials/check-code-details.html.twig:43]: committed suggested

**Comment by SeemaMenon** [comment]
> > If there have been no updated translations provided (i.e. welsh) then there is no need to generate new .po or .mo files as they provide no benefit and may confuse future translation work. Should probably remove those files from this PR.
> 
> @cooperaj - how about the "on %date%" in LPA Viewed field? Thought that needs to be within the translations tag.
> Screenshot 2021-02-26 at 13 35 05



**Removed the po and mo files**

**Review by cooperaj — DISMISSED** [review]
LGTM

**Review by cooperaj — APPROVED** [review]

**Pull request merged**